### ISSUE TITLE
Bezier offseting

### ISSUE THREAD

**Issue by microbians** [jonobr1/two.js#148]
Maybe this could be useful ;)

http://microbians.com/?page=math&id=math-quadraticbezieroffseting

Here is also a demo (but in flash sorry)

http://microbians.com/?page=code&id=code-bezieroffsetingplayground

And here an html5 canvas implementation using my paper: http://brunoimbrizi.com/unbox/2015/03/offset-curve/

**Comment by jonobr1** [comment]
Nice paper and really nice examples! I'm trying to think of typical use cases for this. The main thing that comes to mind for me is to create an abstraction to do varying stroke widths across a path: https://dl.dropboxusercontent.com/u/1595444/experiments/Society%206/trails/index.html

What do you use this type of offsetting for?

**Comment by microbians** [comment]
Yes the process is the same, but is needed to offset for the min and max sizes, and then interpolate the data. I have a sketch app (in flash is old) that works with variable offseting

http://microbians.com/?page=code&id=code-theelectronicsketchbook

**Comment by ceremcem** [comment]
Any progress here? 

I don't know if it helps in any way, here is something I think similar: https://github.com/glenzli/paperjs-offset

**Comment by jonobr1** [comment]
No code committed yet, but this should be straightforward to add. I think this makes sense living in the `/extras` directory. Thanks for the reference!
